src/Symfony/Component/Notifier/Bridge/Teams/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/teams-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Microsoft Teams Notifier Bridge",
+    "keywords": ["teams", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Edouard Lescot",
+            "email": "edouard.lescot@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/notifier": "^5.2"
+    },
+    "autoload": {
+        "psr-4": { "Symfony\\Component\\Notifier\\Bridge\\Teams\\": "" },
```

Comment:
```suggestion
        "psr-4": {
            "Symfony\\Component\\Notifier\\Bridge\\Teams\\": ""
        },
```

Reply:
not really: all the other bridges and components have this on a single line

Reply:
Not all :-) But after this PR: https://github.com/symfony/symfony/pull/39452

😄